src/commands/tick-log.js

```diff
@@ -102,13 +106,18 @@ function writeDefaultMessage(arr) {
       + 'Create some with \'tick commit\'')
 }
 
-function writeEntryGroup(group, hideDetails = false) {
+function writeEntryGroup(group, hideDetails = false, hideSummary = false) {
   const date = moment(group.date).format('ddd, MMM DD, YYYY')
   const duration = format(group.minutes)
-  console.log(`${chalk.yellow(date)} ${duration}`)
 
-  if (!hideDetails) {
+  if (!hideSummary) {
+    console.log(`${chalk.yellow(date)} ${duration}`)
+  }
+
+  if (hideSummary) {
+    group.ticks.forEach(t => console.log(` ${getOutputs(t).detailed}`))
+} else if (!hideDetails) {
```

Comment:
Since we aren't outputting the summary, we don't need to indent the entries. Could you remove the space in from of the detailed output?

Also, could you indent the `else if` so it's inline with the rest of the code?